stable/app/templates/cron.yaml

```diff
@@ -7,7 +7,7 @@
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: "{{ $fullName | trunc 24 }}-{{ $i }}-{{ $job.name | trunc 24}}"
+  name: "{{ $fullName | trunc 29 }}-{{ $i }}-{{ $job.name | trunc 29}}"
```

Comment:
Instead of this, can we add more condition to override the cron name? It should be optional

reason being the k8s pod name have maximum character length

Reply:
Ref https://pauldally.medium.com/why-you-try-to-keep-your-deployment-names-to-47-characters-or-less-1f93a848d34c#:~:text=Pod%20names%2C%20however%2C%20can',without%20the%20dash%2Fhyphen.

Reply:
added condition to trim suffix